src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  return 'hello serhii';
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
+  } else if (days >= 3) {
+    finalPrice = 40 * days - 20;
+  } else {
+    finalPrice = 40 * days;
+  }
+
+  return finalPrice;
```

Comment:
The function `calculateRentalCost` should use constants for the daily cost, the discount for 3 or more days, and the discount for 7 or more days to avoid magic numbers and make the code more maintainable.